test/test_color.py

```diff
@@ -64,6 +69,7 @@ def test_retry_on_server_error():
         assert len(history) == 3
 
 
+@pytest.mark.geoportail
```

Comment:
This test mocks connection. So it does not connect to Geoportail

You can remove the marker `@pytest.mark.geoportail`